datalad_dataverse/tests/test_dataset.py

```diff
@@ -19,30 +19,42 @@ def test_file_handling(
         dataverse_dataaccess_api,
         dataverse_dataset,
 ):
-    # the starting point of `dataverse_dataset` is a freshly
-    # created, non-published dataset in draft mode, with no prior
-    # version
     odd = ODD(dataverse_admin_api, dataverse_dataset)
 
-    fcontent = 'some_content'
-    fpath = tmp_path / 'dummy.txt'
-    fpath.write_text(fcontent)
-    src_md5 = md5sum(fpath)
-
-    fileid = check_upload(odd, fcontent, fpath, src_md5)
-
-    check_download(odd, fileid, tmp_path / 'downloaded.txt', src_md5)
-
-    check_file_metadata_update(dataverse_admin_api, dataverse_dataset, odd,
-                               fileid, fpath)
-
-    fileid = check_replace_file(odd, fileid, tmp_path)
-
-    check_rename_file(odd, fileid)
+    paths = (
+        tmp_path / '.Ö-dot-Ö-in-front' / '!-üö-.txt',
+        tmp_path / '.dot-in-front' / 'c1.txt',
+        tmp_path / ' space-in-front' / 'c2.txt',
+        tmp_path / '-minus-in-front' / 'c3.txt',
+        tmp_path / 'Ö-in-front' / 'überflüssiger_fuß.txt',
+        tmp_path / ' Ö-space-Ö-in-front' / 'a.txt',
+        tmp_path / 'dummy.txt',
+    )
 
-    check_remove(odd, fileid, PurePosixPath(fpath.name))
+    path_info = dict()
+    for path in paths:
+        if path.parent != tmp_path:
```

Comment:
`path.parent.mkdir(parents=True, exist_ok=True)` instead?

Reply:
Used that, thanks